release_notes/369-release-notes.adoc

```diff
@@ -12,59 +12,73 @@ toc::[]
+* *ROX-9400*: Previously, {product-title-short} did not remove the related service accounts when you deleted a cluster. This issue has been fixed.
+* *ROX-9483*: Previously, certain search conditions using a process name could sometimes cause Central to stop responding. This issue has been fixed.
 
 [id="important-system-changes"]
 == Important system changes
 
+* Red Hat has changed the default `grpcPort` in Scanner's configuration map to `8443`.
+* Red Hat is deprecating the following API endpoints:
+
+** `/v1/clusters-env/kernel-support-available`: Use `/v1/cluster-defaults` instead.
+** `/v1/helm/cluster/add`: Use the Helm charts directly.
+** Empty values for `role.access_scope_id` is deprecated in the `RoleService_CreateRole` and `RoleService_UpdateRole` methods for the `/v1/roles/` endpoint. It is now set to the unrestricted access scope ID `io.stackrox.authz.accessscope.unrestricted`.
+
 [id="policy-workflow"]
 === Redesigned policy creation workflow
-{product-title} version 3.69 includes updates  to the policy creation and editing workflows to streamline a user's experience.
+{product-title} 3.69 includes more intuitive and easier to use policy creation and editing workflows.
 
 [id="sorting-and-filtering"]
 === Enhancements to sorting and filtering image vulnerabilities
-{product-title} has added the ability to sort and filter on various data about the vulnerabilities contained within an image.
+{product-title} 3.69 includes new fields for vulnerabilities contained within an image which you use to sort and filter the vulnerabilities list.
 
 [id="uefi-ebpf"]
 === Enhanced compatibility with UEFI secure boot
-Collector is incompatible with UEFI secure boot when collecting runtime data using kernel modules. Now, when the collector detects that UEFI secure boot is used by the host it will automatically fail over to use EBPF probes to prevent service disruption. 
+Because Collector is incompatible with UEFI secure boot when collecting runtime data using kernel modules. In {product-title} 3.69, when Collector detects that the host is using UEFI secure boot, it automatically fails over to use EBPF probes to prevent service disruption.
 
 [id="memory-limit-scanner"]
 === Scanner memory limit increases
-The default Scanner memory limit was increased from 3000 MiB to 4GiB.
+Red Hat has increased the default Scanner memory limit from `3000` MiB to `4` GiB.
 
 [id="deprecation-notices]
-== Deprecation Notices
-The following features have been deprecated and will be removed in version 3.71.0. If you have any questions or concerns please contact support@redhat.com.
+== Deprecation Notice
+Red Hat is deprecating the following features in {product-title} 3.69:
+
+* *External authorization plug-in for scoped access control*. Use the existing in-product scoped access control.
+* *Anchore*, *Tenable*, and *Docker Trusted Registry* integrations. The {product-title-short} scanner supersedes these integrations.
```

Comment:
Should we make it clear we are deprecating the DTR Scanner? I think the integration is both (or either) a Scanner and Registry. I'm not 100% on this maybe @IAmATeaPot418 ?